### DISCUSSION TITLE
3.0.6

### DISCUSSION THREAD

**Discussion by bonanitech** [node-red-contrib-themes/theme-collection#183] [Announcements]
# What’s changed

## 🐛 Bug fixes

* Fix diff colors ([#182](https://github.com/node-red-contrib-themes/theme-collection/pull/182)) @bonanitech
* Fix form input border selected color ([#181](https://github.com/node-red-contrib-themes/theme-collection/pull/181)) @bonanitech
* Fix form input border selected color ([#180](https://github.com/node-red-contrib-themes/theme-collection/pull/180)) @bonanitech
* Fix text colors ([#179](https://github.com/node-red-contrib-themes/theme-collection/pull/179)) @bonanitech

## ⬆️ Dependency updates

* Bump nodemon from 2.0.19 to 2.0.20 ([#177](https://github.com/node-red-contrib-themes/theme-collection/pull/177)) @dependabot
* Bump release-drafter/release-drafter from 5.20.1 to 5.21.0 ([#165](https://github.com/node-red-contrib-themes/theme-collection/pull/165)) @dependabot
* Bump actions/stale from 5.1.1 to 5.2.0 ([#164](https://github.com/node-red-contrib-themes/theme-collection/pull/164)) @dependabot

**Full Changelog**: https://github.com/node-red-contrib-themes/theme-collection/compare/3.0.5...3.0.6

<hr /><em>This discussion was created from the release <a href='https://github.com/node-red-contrib-themes/theme-collection/releases/tag/3.0.6'>3.0.6</a>.</em>

**Comment by Dovahkiin8625** [comment]
**I was trying to build my theme.** 
When I run " npm run dev <theme-name>" ,nothing has been output after 'node ./script/dev.js "aurora"' . And "localhost:41880" can not be accessed .(all developments of node-red and theme-collection has been installed) How can I fix it ?
<img src="http://www.vancode.top:9001/vancode/%E5%BE%AE%E4%BF%A1%E6%88%AA%E5%9B%BE_20221031154524.png"></img>

**Reply by bonanitech** [reply]
What were the steps you followed to get the dev environment installed? Any errors during that process?

**Reply by Dovahkiin8625** [reply]
I excute "yarn" and "yarn run init" in turn . And then I run "yarn run dev" .No error occurred during the above process.
![image](https://user-images.githubusercontent.com/36195882/199165379-a72251c0-126a-4698-81f0-8e034fb857e9.png)
![image](https://user-images.githubusercontent.com/36195882/199166012-368d11ec-5451-41e0-998c-09ac23f27b2e.png)

**Reply by Dovahkiin8625** [reply]
But I can start node-red in theme-collection/node-red folder 
![image](https://user-images.githubusercontent.com/36195882/199166380-cdad2c27-0a08-4f5a-a110-bd2a0a2d2de1.png)

**Reply by bonanitech** [reply]
Please follow [these instructions](https://github.com/node-red-contrib-themes/theme-collection/blob/master/DEVELOPMENT.md)
